src/xo-app/new/import/index.css

```diff
@@ -0,0 +1,32 @@
+.dropzone {
+    border-color: #8f8686;
+    border-radius: 4px;
+    border-style: dashed;
+    border-width: 2px;
+    height: 200px;
+    margin-bottom: 20px;
+    padding: 30px;
+    width: 100%;
+    display: flex;
+}
+
+.activeDropzone {
+    background-color: #f0f0f0;
+    border-style: solid;
+}
+
+.dropzoneText {
+    color: #8f8686;
```

Comment:
You can use [CSS values](https://github.com/tivac/modular-css#values) to avoid repeating values.